query/src/sql/planner/binder/copy.rs

```diff
@@ -445,6 +452,8 @@ impl<'a> Binder {
         let query = self
             .bind_statement(bind_context, &Statement::Query(Box::new(src_query.clone())))
             .await?;
+        // Make sure query has been optimized.
+        let query = optimize(self.ctx.clone(), query)?;
```

Comment:
https://github.com/datafuselabs/databend/blob/c774f800d5e458062c5204d7822668125ff66ba9/query/src/sql/optimizer/mod.rs#L66

IMO it's better to enable optimization for `CopyPlan` here.

Reply:
Addressed.